frontend/app/dashboard/src/views/dashboard/settings/modules/members/ImportMembersQuestionsView.vue

```diff
@@ -172,467 +214,412 @@
+}
 
-@Component({
-    components: {
-        STNavigationBar,
-        STToolbar,
-        STInputBox,
-        RadioGroup,
-        Radio,
-        STErrorsDefault,
-        Checkbox,
-        BackButton,
-        LoadingButton,
-        STList,
-        STListItem,
-        Dropdown
+onMounted(() => {
+    multipleGroups.value = calculateMultipleGroups()
+    autoAssignMembers($members.value);
+});
+
+const organizationPatch: AutoEncoderPatchType<Organization> & AutoEncoder = OrganizationPatch.create({});
+const organization = computed(() => $organization.value!.patch(organizationPatch));
+const errorBox: Ref<ErrorBox | null> = ref(null);
+const saving = ref(false);
+const paid = ref<boolean | null>(true);
+const waitingList = ref(false);
+const needRegistration = ref(false);
+const autoAssign = ref(true);
+const multipleGroups: Ref<Group[]> = ref([]);
+const draggableGroups = computed({
+    get() {
+        return multipleGroups.value;
     },
-})
-export default class ImportMembersQuestionsView extends Mixins(NavigationMixin) {
-    errorBox: ErrorBox | null = null
-    validator = new Validator()
-    saving = false
-    organizationPatch: AutoEncoderPatchType<Organization> & AutoEncoder = OrganizationPatch.create({})
-
-    @Prop({ required: true })
-        members: ImportingMember[]
-
-    created() {
-        this.organizationPatch.id = this.$organization.id
-    }
-
-    paid: boolean | null = true
-    waitingList = false
-    needRegistration = false
-
-    autoAssign = true
-    multipleGroups: Group[] = []
-    defaultGroup = this.organization.groups[0]
-
-    mounted() {
-        this.multipleGroups = this.calculateMultipleGroups()
-        this.autoAssignMembers(this.members)
-    }
+    set(groups: Group[]) {
+        if (groups.length != multipleGroups.value.length) {
+            return;
+        }
 
-    get organization() {
-        return this.$organization.patch(this.organizationPatch)
+        multipleGroups.value = groups;
     }
+});
 
-    get needsPaidStatus() {
-        return !!this.membersWithNewRegistrations.find(m => m.registration.paid === null && m.registration.paidPrice === null)
-    }
+const defaultGroup = ref(organization.value.groups[0]) as Ref<Group>;
 
-    get somePaid() {
-        return !!this.membersWithNewRegistrations.find(m => m.registration.paid !== null || m.registration.paidPrice !== null)
-    }
+//#region created
+organizationPatch.id = organization.value.id
+//#endregion
 
-    get needsGroupAssignment() {
-        return this.membersNeedingAssignment.length > 0
+const membersWithNewRegistrations = computed(() => {
+    return $members.value.filter(m => {
+        const registration = buildRegistration(m)
+        if (!registration) {
+            return false
+        }
+        return true;
+    })
+});
+
+const needsPaidStatus = computed(() => Boolean(membersWithNewRegistrations.value.find(m => m.registration.paid === null && m.registration.paidPrice === null)));
```

Comment:
```suggestion
const needsPaidStatus = computed(() => !!membersWithNewRegistrations.value.find(m => m.registration.paid === null && m.registration.paidPrice === null));
```

Kan je dit herschrijven naar !! in plaats van Boolean?